### DISCUSSION TITLE
Fehlermeldungen vom Wechselrichter

### DISCUSSION THREAD

**Discussion by stumpf001** [tbnobody/OpenDTU#452] [General]
Hallo,

kann man mit der openDTU auch event. auftretende Fehlermeldungen vom HM-XXX auslesen? Oder ist dies (noch) nicht möglich?

Gruß Volker

**Comment by tbnobody** [comment]
Du meinst das Eventlog? https://github.com/tbnobody/OpenDTU/blob/master/docs/screenshots/12_Eventlog.png

**Comment by stumpf001** [comment]
Ja, wenn hier auch Fehlermeldungen vom Wechselrichter gelogt werden, dann ist das, das ich gemeint habe, danke.

**Comment by tbnobody** [comment]
Auf das, was dort genau gelogged wird habe ich keinen Einfluss. Das kommt 1zu1 so aus dem Inverter. Aber die bereits entschlüsselten Fehlernummern deuten definitiv auch auf Fehlermeldungen hin: https://github.com/tbnobody/OpenDTU/blob/9b0ab67849c4d93d6d7d96a34a1d6fcd3119bf65/lib/Hoymiles/src/parser/AlarmLogParser.cpp#L60-L271

**Reply by stumpf001** [reply]
Ja, das sieht gut aus. Aber man hofft natürlich, dass man die Fehlermeldungen nicht zu sehen bekommt ;)

**Reply by Beathunter1** [reply]
![eventlog](https://user-images.githubusercontent.com/130597332/231804510-b5948aaa-b262-41db-903b-ec9e25e24543.PNG)

Hat jemand eine Idee was Fehler 46 sein soll???

**Reply by DPO99** [reply]
Den hab ich auch ab und an mal, wenn ich die Kiste neu starte...würde mich auch interessieren :D

**Reply by tbnobody** [reply]
Glaubt mir, wenns in einem Dokument stehen würde oder schon bekannt wäre, würde da nicht "Unknown" stehen :)

Falls jemand tipps hat, gerne her damit! Aber bisher tut man sich mit den 2 stelligen Events sehr schwer da etwas herauszubekommen.

**Reply by tbnobody** [reply]
@Beathunter1 so interessehalber, welche OpenDTU Version verwendest du?

**Reply by Beathunter1** [reply]
Die letzte

**Reply by Beathunter1** [reply]
414 glaube ich

**Reply by tbnobody** [reply]
414 ist eigentlich keine Versionsnummer.... man findet die unter Info --> System --> "Firmwareversion / git Hash"

**Reply by Beathunter1** [reply]
Bor, du fragst Sachen 😂 hab vorgestern das letzte Package installiert

**Comment by themanfrommoon** [comment]
Weiß jemand was Fehlercode 2 und Fehlercode 46 bedeutet?
![Screenshot_20230416_193456_Samsung Internet](https://user-images.githubusercontent.com/28815063/232330506-e64ad0fe-bfdd-46c1-b5dd-d07da13cb575.jpg)

**Reply by DPO99** [reply]
Na 2 steht ja da. 46 ist nicht bekannt, daher unknown.

**Reply by themanfrommoon** [reply]
....ja, steht da: DTU command failed.
Aber was soll mir das jetzt sagen?
Welcher Befehl ist fehlgeschlagen, und warum? (ich habe gar keinen Befehl bewusst gesendet)
Ich habe 3 Stück HM-600 in Betrieb.
Nur einer haut alle paar Sekunden den Fehlercode raus.
Das muss ja irgendeinen Grund haben, und den würde ich gerne abstellen.

**Reply by Beathunter1** [reply]
Hast du mal versucht die Position des Transceivers zu ändern? Vielleicht hängt ja ein HM-600er im Funkschatten oder so…

**Reply by themanfrommoon** [reply]
Das probiere ich mal aus.
Entfernung ist ca 50m Luftlinie, ohne großartige Hindernisse dazwischen. Ich hab das ebyte Modul und die Transmitting Power auf Maximum gestellt. Ich habe allerdings keinen Puffer Elko verbaut. Aber wenn es ein Problem geben würde, dann doch bei beiden Wechselrichtern die auf der DTU laufen (der dritte Wechselrichter läuft auf einer zweiten DTU)
